### ISSUE TITLE
Apple Voiceover fails on the sample chatbot client

### ISSUE THREAD

**Issue by bronius** [Orange-OpenSource/a11y-guidelines#75]
Hi - 
What a terrific resource this is! 

In particular, I am exploring your proposed markup in your "sample chatbot" demo: Works flawlessly in NVDA on Windows, but Apple Voiceover
* starts out by repeating everything in the container (maybe even repeating _twice_, I'm not sure there)
* then seems to work Ok
* then very soon starts gobbling the spoken "Djingo says" aria prefixes as well as, "me says,"
* and eventually gets where it doesn't read any responses but what I wrote

Has anyone observed the same behavior? Know the cause? Surely it isn't _this markup_, but I thought I might start here to find a sympathetic hear and poll for solutions.. Maybe artificial delays would help? Maybe it's unique to me and my system?

Thanks :)
-Bronius

**Comment by GaetanBt** [comment]
Hey ! Thanks for your message, it's truly appreciated :slightly_smiling_face: 

As I'm not working on this part specifically and I don't have any Mac around me to test it, I'm going to share your issue in the team, we'll see if  we can reproduce this behavior and find a way to prevent it

**Comment by bronius** [comment]
Ok thank you, I appreciate your responses :)

I am referring to the Chatbot from the click of the "Sample interaction" button on https://a11y-guidelines.orange.com/en/web/components-examples/chatbot/ . Just now I wanted to vet it with more info:

1. With Apply Voiceover enabled (MacOS Catalina 10.15.7)
1. I click the Sample interaction button
1. I listen to Voiceover read the entire chat conversation once through then it read itself again
1. I click Web
1. I see Djingo's response but didn't hear it read. . . 
1. Then I just typed nonsense like "web", "web web", "web web web" a few lines, and each time the interaction was as expected.
1. Until I typed, "I don't see any issues today" which showed me Djingo's response, but the sound was just a creepy, breathy "Hhhahhhh" wisp of the wind sound.
1. Anything I type after this I get that weird sound.
1. I tried stopping Voiceover and launching it again... same exhaling-breath-like responses only.

In our own Chatbot that we are working on, I have experimented with dynamically aria-busy={true|false} between messages. Didn't seem to do anything ¯\_(ツ)_/¯ Our original Chatbot codebase works quite well in all screenreaders we test with, but it has aria-live="polite" around _each_ new message (this is within a role="log" wrapper) as well as an additional "role=alert" on the first message out the gate. I thought that because our app is React that maybe the aria attributes must be on the innermost, non-dynamic wrapping html element, but that didn't do squat to improve my situation.

hth.

**Comment by PigeardSylvain** [comment]
Thanks for your messages @bronius, you can check chatbot sample. The bug with voiceover has been fixed 6416cbfdb1d8e3be9006256c9b3e7c8c56362ec9

**Comment by bronius** [comment]
> Thanks for your messages @bronius, you can check chatbot sample. The bug with voiceover has been fixed [6416cbf](https://github.com/Orange-OpenSource/a11y-guidelines/commit/6416cbfdb1d8e3be9006256c9b3e7c8c56362ec9)

Thanks very much, @PigeardSylvain - I'll pass this along to the team! 🙂
